### PR TITLE
re-add old swift api examples

### LINKED ISSUE #1604: Bring back old swift examples

### PR DESCRIPTION
**Pull request by alazier**
Closes #1604 

@jpsim @segiddins

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -444,17 +444,28 @@ case "$COMMAND" in
...
         xc "-project examples/ios/objc/RealmExamples.xcodeproj -scheme Encryption -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
-        xc "-project examples/ios/objc/RealmExamples.xcodeproj -scheme Extension -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
-        exit 0
-        ;;
-
-    "examples-ios-swift")
-        xc "-project examples/ios/swift/RealmExamples.xcodeproj -scheme Simple -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
+        
+	# FIXME - re-enable once this doesn't fail for machines without the tightdb cert
+	#xc "-project examples/ios/objc/RealmExamples.xcodeproj -scheme Extension -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
```
Comment (jpsim) [inline build.sh]: How about just this for now?

```sh
if [ -z "${JENKINS_HOME}" ]; then
  xc "-project examples/ios/objc/RealmExamples.xcodeproj -scheme Extension -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
fi
```

Comment (alazier) [inline build.sh]: i think that is the opposite of what we want.

Comment (jpsim) [inline build.sh]: right. should be this instead:

```sh
if ! [ -z "${JENKINS_HOME}" ]; then
  xc "-project examples/ios/objc/RealmExamples.xcodeproj -scheme Extension -configuration $CONFIGURATION build ${CODESIGN_PARAMS}"
fi
```

**Comment by jpsim** [comment]
:+1: if CI passes

**Pull request merged**